python3/amplify/README.md

```diff
@@ -0,0 +1,107 @@
+# Amplify Access Logs Shipper - Lambda
```

Comment:
Please have @nico-shishkin review this README, and add to the docs as well.

Reply:
@nico-shishkin  can you go over of Readme.